src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int HUNDRED = 100;
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        return new Ball(supplier.getRandomColor(),random.nextInt(HUNDRED));
```

Comment:
Read check style of code 
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/e669a8f0-ffbe-4526-ad41-50519f1b1f68)